app/assets/stylesheets/components/common/_travel_insurance_wizard_form.scss

```diff
@@ -0,0 +1,20 @@
+.form.js-wizard{
+  position: relative;
+  overflow: hidden;
```

Comment:
I don't think this is strictly necessary since the items are not shown when not in an "active" state. It also means there is no need for the padding value below to avoid the cut-off of the radio element.